Misc/NEWS.d/next/Library/2023-05-15-17-22-53.gh-issue-104306.YMiegg.rst

```diff
@@ -0,0 +1 @@
+Fix incorrect comment parsing in the ``netrc`` module
```

Comment:
We can turn `netrc` into a link to https://docs.python.org/3/library/netrc.html:

```suggestion
Fix incorrect comment parsing in the :mod:`netrc` module.
```

Reply:
Oh all right didn't know that. Thanks 😁